CPT331.iOS/CPT331.iOS/LocationManager.swift

```diff
@@ -7,23 +7,31 @@
 //
 
 import Foundation
-import CoreData
-import CoreLocation
 import Mapbox
 import MapboxGeocoder
 
+/// Wrapper for the  Mapbox Geocoding API, enables searching for locatons (forward geocoding), and getting location info for a specific coordinate (reverse geocoding).
 class LocationManager {
     static let sharedInstance = LocationManager()
     
-    // Stores a referance to the most recent search task
-    //      To be cancelled if a new task starts
+    /// Stores a referance to the most recent search task. To be cancelled if a new task starts.
     private var lastSearchTask: NSURLSessionDataTask?
     
-    // Prevent external initialization
+    /// Singleton initializer
```

Comment:
Probably don't need this level of detail in the comments. Btw, what's the difference between double and triple slashes? Seems as though there's a sprinkling of the two throughout?